src/examples/src/config.ts

```diff
@@ -209,6 +217,52 @@ export const config: Config = {
 			}
 		]
 	},
+	chip: {
+		filename: 'index',
+		overview: {
+			example: {
+				module: BasicChip,
+				filename: 'Basic'
+			}
+		},
+		examples: [
+			{
+				title: 'Icon',
+				module: IconChip,
+				filename: 'Icon'
+			},
+			{
+				title: 'Clickable',
+				module: ClickableChip,
+				filename: 'Clickable'
+			},
+			{
```

Comment:
Do you think that togglechip / closeablechip etc should be showcased here or split into their own opinionated variant?

Reply:
Toggling seems different enough to merit its own variant, mainly because it might want to add classes depending on the toggle state. I can't think of a good reason to creating a separate widget just for the `onClose` property.